recipes/buffer-manage

```diff
@@ -0,0 +1 @@
+(buffer-manage :fetcher github :repo "plandes/buffer-manage" :files ("*.el"))
```

Comment:
It is not necessary to specify `:files` attribute because it is enough to use default value. Please see https://github.com/melpa/melpa#recipe-format